### PR TITLE
Code Ocean

### PR DESCRIPTION
**Pull request by chafdev**

### CODE REVIEW

**Comment by chafdev** [comment]
exemple :
--static-batch --output C:\Users\chafik.meniar\Documents\test\exportStatic\
pour faire un test local pour code ocean :
 --codeocean --input C:\Users\chafik.meniar\Documents\test\ --output C:\Users\chafik.meniar\Documents\test\test2\
sinon 
--codeocean

**Comment by chafdev** [comment]
il faut mettre à jour la lib mtnp

File: Source/config.xml [hunk 1]
```diff
@@ -108,5 +108,27 @@
 </files>
 	<bundle>true</bundle>
     </dependency>
+    <dependency>
+			<!-- GRAL. Stand-alone plotting library -->
```
Comment (sylvainhalle) [inline Source/config.xml]: The comment does not match the dependency

File: Source/Core/src/ca/uqac/lif/labpal/server/ExplainImageCallback.java [hunk 2]
```diff
@@ -22,106 +22,97 @@
...
-	
-	/*	Set<Integer> ids = m_lab.getTableIds();
-		for (int id : ids) {
-			Table tab = m_lab.getTable(id);
-			HardTable tbl = tab.getDataTable();
-			render1(zos,tab, tbl.getTree(), tbl.getColumnNames());
-			zos.closeEntry();
-		}*/
-		
+
```
Comment (sylvainhalle) [inline Source/Core/src/ca/uqac/lif/labpal/server/ExplainImageCallback.java]: The indentation for this commented bit of code has been completely messed up. Please restore it to what it was.

**Pull request closed without merging**